cloudinstall/controllers/installbase.py

```diff
@@ -159,5 +162,7 @@ def start(self):
             self.ui.select_install_type(
                 self.config.install_types(), self._set_install_type)
 
-        self.update()
+        self.alarm = self.loop.set_alarm_in(1, self.update)
+        self.observe('stop alarm', partial(self.loop.remove_alarm,
+                                           self.alarm))
```

Comment:
ah, the only way i could remove the alarm was assigning it, i need to look at self.update again it may be we just need to refactor that routine some more